internal/mux/mux_test.go

```diff
@@ -1,14 +1,23 @@
 package mux
 
 import (
+	"io"
 	"net"
 	"testing"
 	"time"
 
 	"github.com/pion/logging"
+	"github.com/pion/transport/packetio"
 	"github.com/pion/transport/test"
+	"github.com/stretchr/testify/assert"
 )
 
+const testPipeBufferSize = 8192
+
+func matchAll([]byte) bool {
+	return true
+}
+
```

Comment:
Should matchAll be moved into the `mux` package, and exposed? seems like something people will want to do often...

Reply:
Fixed! I deleted this and brought back the other function.